### PR TITLE
Add TAA kernel parameter testcase

### PR DESCRIPTION
**Pull request by wjn740**
- Verification run: http://10.67.134.217/tests/6657

### CODE REVIEW

**Comment by wjn740** [comment]
@badboywj @skliu @simonlm @tonyyuan1

**Comment by wjn740** [comment]
@dzedro HI sir, would you please help to review this and merge?

File: tests/cpu_bugs/taa.pm [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+# Maintainer: James Wang <jnwang@suse.com>
+
+package taa;
+use strict;
+use warnings;
+use base "consoletest";
+use base "Mitigation";
+use bootloader_setup;
+use ipmi_backend_utils;
+use power_action_utils 'power_action';
```
Comment (dzedro) [inline tests/cpu_bugs/taa.pm]: You don't use power_action here

Comment (wjn740) [reply tests/cpu_bugs/taa.pm]: line 67, call reboot_and_wait, but I don't need import 'power_aciton' here, right?

Comment (dzedro) [reply tests/cpu_bugs/taa.pm]: right

**Pull request merged**